segregation/aspatial/__init__.py

```diff
@@ -0,0 +1 @@
+from .non_spatial_indexes import *
```

Comment:
I think we can go all the way to `aspatial`, can you rename this file and also this line to `aspatial_indexes`?

Reply:
This is the only change.